OMCompiler/Compiler/Script/OpenModelicaScriptingAPI.mos

```diff
@@ -6,6 +6,7 @@ if bool then
   bool := loadFile("OpenModelicaScriptingAPI.tmp.mo");
   if bool then
     bool := OpenModelica.Scripting.compareFilesAndMove("OpenModelicaScriptingAPI.tmp.mo", "OpenModelicaScriptingAPI.mo");
+    OpenModelica.Scripting.remove("OpenModelicaScriptingAPI.tmp.mo");
```

Comment:
I just checked the definition for `remove` and it looks like it is intended for removing directories. I am not sure what happens when the argument is a file.

https://github.com/OpenModelica/OpenModelica/blob/2cbc275421af56c4c6bf8451e92f0e4bbe795c0f/OMCompiler/Compiler/Script/CevalScript.mo#L1045-L1046

Reply:
```suggestion
    OpenModelica.Scripting.removeFile("OpenModelicaScriptingAPI.tmp.mo");
```